persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -383,7 +383,7 @@ ORDER BY Conversation.type DESC LIMIT 1;
 getMLSGroupIdByUserId:
 SELECT Conversation.mls_group_id FROM Member
 JOIN Conversation ON Conversation.qualified_id = Member.conversation
-WHERE Conversation.mls_group_id IS NOT NULL AND Member.user = :userId;
+WHERE Conversation.mls_group_id IS NOT NULL AND Member.user = :userId LIMIT 1;
```

Comment:
is there a case where there is more than 1 conversation with the same mls_group_id ?

Reply:
I think since the member is the source, can be more than one conversation that the user is member of!